test/render-utils.test.js

```diff
@@ -0,0 +1,16 @@
+
+import { renderItem } from '../render-utils.js';
+
+const test = QUnit.test;
+
+test('time to render item', (expect) => {
+    
+    const expected = '<div class="incomplete items"><p>spice</p></div>';
+    
+    //Act 
+    // Call the function you're testing and set the result to a const
+    const actual = renderItem({ complete: false, items:'spice' }); 
+
+    
+    expect.equal(actual.outerHTML, expected);
+});
```

Comment:
Excellent job testing your PURE function! Another test that would be easy to add would be to add a complete=true within the renderItems function and ensure that our class updates properly. Nice work Andy!